### PR TITLE
AKV 2.0.0 release notes

### PR DESCRIPTION
**Pull request by cheenamalhotra**

### CODE REVIEW

File: release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Release Notes
+
+## General Availability of Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyVaultProvider
+_**2.0.0 released 03 March 2021**_
+
+This library contains the implementation of `Microsoft.Data.SqlClient.SqlColumnEncryptionKeyStoreProvider` for accessing Azure Key Vault, and the provider class is named `SqlColumnEncryptionAzureKeyVaultProvider`.
+
+### Added
+- Upgraded AKV Provider to use new Azure Key Vault libraries [#930](https://github.com/dotnet/SqlClient/pull/630)
```
Comment (johnnypham) [inline release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md:9]: Shouldn't use "AKV" in public docs, not sure if it applies to release notes.

Comment (johnnypham) [inline release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md:9]:
```suggestion
- Upgraded AKV Provider to use new Azure Key Vault libraries [#630](https://github.com/dotnet/SqlClient/pull/630)
```

File: release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+### Breaking Changes
+- Drops support for .NET Framework 4.6. The new minimum supported .NET Framework version is v4.6.1 [#930](https://github.com/dotnet/SqlClient/pull/630)
```
Comment (johnnypham) [inline release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md:12]:
```suggestion
- Drops support for .NET Framework 4.6. The new minimum supported .NET Framework version is v4.6.1 [#630](https://github.com/dotnet/SqlClient/pull/630)
```

This should be #630 too?

File: release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+- Updated dependency Mcrosoft.Data.SqlClient on .NET Framework and .NET Core to LTS stable version v1.1.3+ [#946](https://github.com/dotnet/SqlClient/pull/946)
+
+
+### Working with SQLColumnEncryptionAzureKeyVaultProvider
+`SqlColumnEncryptionAzureKeyVaultProvider` is implemented against `Microsoft.Data.SqlClient` and supports .NET Framework 4.6.1+, .NET Core 2.1+, and .NET Standard 2.0+. The provider name identifier for this library is "**AZURE_KEY_VAULT**" and it is not registered in the driver by default. Client applications may call the `SqlConnection.RegisterColumnEncryptionKeyStoreProviders()` API once in the lifetime of the driver to register this custom provider by implementing a custom Authentication Callback mechanism.
```
Comment (johnnypham) [inline release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md:17]: New provider doesn't use callbacks.

File: release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md (hunk 1 above)
Comment (cheenamalhotra) [reply release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md:9]: Could you also include abbreviation removal here in suggestion - let's fix that too.

Comment (David-Engel) [reply release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md]: @johnnypham is correct.

File: release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+_**2.0.0 released 03 March 2021**_
+
+This library contains the implementation of `Microsoft.Data.SqlClient.SqlColumnEncryptionKeyStoreProvider` for accessing Azure Key Vault, and the provider class is named `SqlColumnEncryptionAzureKeyVaultProvider`.
+
+### Added
+- Upgraded AKV Provider to use new Azure Key Vault libraries [#930](https://github.com/dotnet/SqlClient/pull/630)
+
+### Breaking Changes
+- Drops support for .NET Framework 4.6. The new minimum supported .NET Framework version is v4.6.1 [#930](https://github.com/dotnet/SqlClient/pull/630)
+- Updated dependency Mcrosoft.Data.SqlClient on .NET Framework and .NET Core to LTS stable version v1.1.3+ [#946](https://github.com/dotnet/SqlClient/pull/946)
```
Comment (David-Engel) [inline release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md:13]:
```suggestion
- Updated dependency of Microsoft.Data.SqlClient on .NET Framework and .NET Core to LTS stable version v1.1.3+ [#946](https://github.com/dotnet/SqlClient/pull/946)
```

File: release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md (hunk 3 above)
Comment (cheenamalhotra) [reply release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md]: Fixed

File: release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+
+### Working with SQLColumnEncryptionAzureKeyVaultProvider
+`SqlColumnEncryptionAzureKeyVaultProvider` is implemented against `Microsoft.Data.SqlClient` and supports .NET Framework 4.6.1+, .NET Core 2.1+, and .NET Standard 2.0+. The provider name identifier for this library is "**AZURE_KEY_VAULT**" and it is not registered in the driver by default. Client applications may call the `SqlConnection.RegisterColumnEncryptionKeyStoreProviders()` API once in the lifetime of the driver to register this custom provider by implementing a custom Authentication Callback mechanism.
+
+Once the provider is registered, it can be used to perform Always Encrypted operations by creating a Column Master Key using the Azure Key Vault Key Identifier URL.
+
+Below linked C# samples demonstrate using Always Encrypted and Secure Enclaves with Azure Key Vault:
```
Comment (David-Engel) [inline release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md:21]:
```suggestion
The linked C# samples below demonstrate using Always Encrypted and Secure Enclaves with Azure Key Vault:
```

File: release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+- Drops support for .NET Framework 4.6. The new minimum supported .NET Framework version is v4.6.1 [#630](https://github.com/dotnet/SqlClient/pull/630)
+- Updated dependency of Microsoft.Data.SqlClient on .NET Framework and .NET Core to LTS stable version v1.1.3+ [#946](https://github.com/dotnet/SqlClient/pull/946)
+
+
+### Working with SQLColumnEncryptionAzureKeyVaultProvider
+`SqlColumnEncryptionAzureKeyVaultProvider` is implemented against `Microsoft.Data.SqlClient` and supports .NET Framework 4.6.1+, .NET Core 2.1+, and .NET Standard 2.0+. The provider name identifier for this library is "**AZURE_KEY_VAULT**" and it is not registered in the driver by default. Client applications may now initialize this provider by providing an instance of `Azure.Core.TokenCredential` implementation and register it with the driver.
+
+Once the provider is registered, it can be used to perform Always Encrypted operations by creating a Column Master Key using the Azure Key Vault Key Identifier URL.
+
+The linked C# samples below demonstrate using Always Encrypted and Secure Enclaves with Azure Key Vault:
```
Comment (David-Engel) [inline release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md:21]: I missed this in my first pass. "secure enclaves" should be lower case.
```suggestion
The linked C# samples below demonstrate using Always Encrypted with secure enclaves with Azure Key Vault:
```

File: release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+- Legacy API support (Always Encrypted): [AzureKeyVaultProviderExample.cs](https://github.com/dotnet/SqlClient/blob/master/doc/samples/AzureKeyVaultProviderLegacyExample_2_0.cs)
+- New API support (Always Encrypted): [AzureKeyVaultProviderExample.cs](https://github.com/dotnet/SqlClient/blob/master/doc/samples/AzureKeyVaultProviderExample_2_0.cs)
+- Legacy API support (Always Encrypted with Secure Enclaves): [AzureKeyVaultProviderExample.cs](https://github.com/dotnet/SqlClient/blob/master/doc/samples/doc\samples\AzureKeyVaultProviderWithEnclaveProviderExample.cs)
```
Comment (David-Engel) [inline release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md:24]:
```suggestion
- Legacy API support (Always Encrypted with secure enclaves): [AzureKeyVaultProviderExample.cs](https://github.com/dotnet/SqlClient/blob/master/doc/samples/doc\samples\AzureKeyVaultProviderWithEnclaveProviderExample.cs)
```

File: release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+- New API support (Always Encrypted with Secure Enclaves): [AzureKeyVaultProviderExample.cs](https://github.com/dotnet/SqlClient/blob/master/doc/samples/doc\samples\AzureKeyVaultProviderWithEnclaveProviderExample_2_0.cs)
```
Comment (David-Engel) [inline release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md:25]:
```suggestion
- New API support (Always Encrypted with secure snclaves): [AzureKeyVaultProviderExample.cs](https://github.com/dotnet/SqlClient/blob/master/doc/samples/doc\samples\AzureKeyVaultProviderWithEnclaveProviderExample_2_0.cs)
```

**Review by David-Engel — APPROVED** [review]

**Pull request merged**